angularjs-datetime-picker.js

```diff
@@ -34,7 +34,7 @@
     var datetimePickerEl;
     var _this = this;
     var removeEl = function(el) {
-      el && el.remove();
+      el && el.parentElement.removeChild(el);
```

Comment:
Because IE doesn't recognize function remove and throw error with message "Object doesn't support property or method 'remove'". More details here: https://developer.mozilla.org/en-US/docs/Web/API/ChildNode/remove